vispy/scene/cameras/cameras.py

```diff
@@ -151,60 +428,61 @@ def _set_scene_transform(self, tr):
         else:
             self._transform_cache.roll()
             self._scene_transform = self._transform_cache.get([pre_tr, tr])
-            
-        if self.viewbox is not None:
-            self.viewbox.scene.transform = self._scene_transform
-            self.viewbox.update()
+        
+        # Update scene    
+        self._viewbox.scene.transform = self._scene_transform
+        self._viewbox.update()
+        
+        # Apply same state to linked cameras, but prevent that camera
+        # to return the favor
+        if not self._linked_cameras_no_update:
+            for cam in self._linked_cameras:
+                cam._linked_cameras_no_update = True
+                try:
+                    cam.set_state(self.get_state())
+                finally:
+                    cam._linked_cameras_no_update = False
 
     
 class PanZoomCamera(BaseCamera):
-    """
-    Camera implementing 2D pan/zoom mouse interaction. Primarily intended for
-    displaying plot data.
-
+    """ Camera implementing 2D pan/zoom mouse interaction.
+    
+    For this camera, the ``scale_factor`` indicates the zoom level, and
+    the ``center`` indicates the center position of the view.
+    
     By default, this camera inverts the y axis of the scene. This usually 
     results in the scene +y axis pointing upward because widgets (including 
     ViewBox) have their +y axis pointing downward.
     
-    User interaction:
-    
-    * Dragging left mouse button pans the view
-    * Dragging right mouse button vertically zooms the view y-axis
-    * Dragging right mouse button horizontally zooms the view x-axis
-    * Mouse wheel zooms both view axes equally.
-
     Parameters
     ----------
-    parent : Node
-        The parent of the camera.
-    name : str
-        Name used to identify the camera in the scene.
+    See BaseCamera.
+    
+    Interaction
+    -----------
+    * LMB: pan the view
+    * RMB or scroll: zooms the view
+    
     """
     def __init__(self, **kwargs):
         super(PanZoomCamera, self).__init__(**kwargs)
-        self._rect = Rect((0, 0), (1, 1))  # visible range in scene
-        self._invert = [False, True]
+        # todo: Y axis is flipped by default?
+        #self._aspect_ratio = self._aspect_ratio_n = 1.0, -1.0, 0.0
         self.transform = STTransform()
-        
-    def zoom(self, zoom, center):
-        """ Zoom the view around a center point.
+    
+    def zoom(self, scale_factor, center):
```

Comment:
I am not sure if I agree. The function looks to me as if calling it twice will yield the same result. I think because of the presense of the `center` parameter. You suggested a "function to allow relative zoom" earlier, but because of the presense of this function I decided that people could just do `camera.scale_factor *= 2.0`. If you know another solution I'd be happy to hear it.